helm_deploy/values-preprod.yaml

```diff
@@ -18,6 +18,11 @@ generic-service:
     MRD_URL: https://make-recall-decision-preprod.hmpps.service.justice.gov.uk
     MRD_API_URL: https://make-recall-decision-api-preprod.hmpps.service.justice.gov.uk
     HOUSEKEEPING_SENDDOMAINEVENTS: true
+    # we cannot override a single item in a list - the whole list is replaced, so we define the whole thing again
```

Comment:
These overrides use the old structure and document names and need updating (and the preview overrides removed, I think?).